x-pack/plugins/watcher/public/application/sections/watch_list_page/watch_list_page.tsx

```diff
@@ -178,24 +177,20 @@ export const WatchListPage = () => {
 
   if (isWatchesLoading) {
     return (
-      <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
+      <EuiPageTemplate.EmptyPrompt>
```

Comment:
Added `EuiPageTemplate.EmptyPrompt` otherwise the loading section doesn't have the color. 
<img width="360" alt="Screenshot 2023-08-22 at 13 36 41" src="https://github.com/elastic/kibana/assets/6585477/9a3a42c1-82f6-4e1c-aed0-bb9dbf958ee2">